components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -526,7 +571,7 @@ - (void)showLargeItemDialog {
   [self.largeItemDialog.centerYAnchor constraintEqualToAnchor:window.centerYAnchor].active = YES;
 
   self.largeItemDialog.layer.opacity = 0;
-  self.largeItemDialog.transform = MDCLargeItemViewAnimationTransitionTransform();
+  self.largeItemDialog.transform = LargeItemViewAnimationTransitionTransform();
```

Comment:
Nit: Let's move this refactoring to a separate PR in case we need to roll-back the state restoration change.

Reply:
Done.